### PR TITLE
Update openoffice.txt

### PR DESCRIPTION
**Pull request by Pilot-Pirx**
Update to version 4.1.10

### CODE REVIEW

File: openoffice.txt
Comment (learn-more) [inline openoffice.txt:6]: Size is not needed if SizeBytes is present.

Comment (Pilot-Pirx) [reply openoffice.txt:6]: OK, I just updated the old file...
SizeBytes is preferred?

File: openoffice.txt
Comment (learn-more) [inline openoffice.txt:75]: `ua` is not `uk`, is this an intentional change?

Comment (Pilot-Pirx) [reply openoffice.txt:75]: Yes, uk is the language code for Ukrainian. ua is the country code for the Ukraine.
OpenOffice only supports languages, not countries.
That was fixed on the homepage.

Comment (binarymaster) [reply openoffice.txt:75]: Confirming, `/ua/` leads to 404 page.

**Review by blackcrack — APPROVED** [review]

**Review by learn-more — APPROVED** [review]

**Comment by Pilot-Pirx** [comment]
"1 workflow awaiting approval
First-time contributors need a maintainer to approve running workflows"

Any idea when this can get merged?

**Pull request merged**
